src/services/orders.ts

```diff
@@ -23,8 +23,11 @@ export const getAllOrdersAsUIOrders = async (baseToken: Token, quoteToken: Token
     const orders: SignedOrder[] = await getAllOrders(baseToken, quoteToken);
     try {
         const contractWrappers = await getContractWrappers();
-        const ordersInfo = await contractWrappers.exchange.getOrdersInfoAsync(orders);
-        return ordersToUIOrders(orders, baseToken, ordersInfo);
+        const ordersAndTradersInfo = await contractWrappers.orderValidator.getOrdersAndTradersInfoAsync(
```

Comment:
This version of `OrderValidator` does not support MAP orders, and also does not validate signatures, FYI.